changelog/changelog2022.md

```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
```

Comment:
## 26th September 2022

* In [Process Group](../mews-operations/reservations.md#process-group), `channel` is deprecated and replaced by `sources`, which is a collection of new object [Source](../mews-operations/reservations.md#source) and provides additional information about reservation sources or channels.
* In [Process Group](../mews-operations/reservations.md#process-group), `company` is split into separate entities for [Company](../mews-operations/reservations.md#company) and [Travel Agency](../mews-operations/reservations.md#travel-agency). `iata` is removed from `company` (deprecated).